internal/jimm/jimm.go

```diff
@@ -195,9 +196,8 @@ type SSHManager interface {
 	// PublicKeyHandler is the method to verify the public key of the user. It returns a user if successful.
 	PublicKeyHandler(ctx context.Context, claimUser string, key []byte) (*openfga.User, error)
 
-	// ControllerInfoFromModelUUID is the method to resolve the address of the controller to contact given the model UUID and
-	// a valid JWT To connect to the controller.
-	ControllerInfoFromModelUUID(ctx context.Context, modelUUID string, user *openfga.User) (ssh.ControllerInfo, error)
+	// DialControllerSSHServer dials the controller hosting the specific model UUID.
```

Comment:
This is worded weirdly... Uhm maybe something like "dials the hosting controller's SSH server for this model."